requirements.txt

```diff
@@ -4,6 +4,7 @@ pytest >= 6.2.0
 pytest-cov >= 2.4.0
 pytest-localserver >= 0.4.1
 
+aiohttp == 3.8.1
```

Comment:
Is there a reason to pin the version to `3.8.1` or is it safer to to `>=` here as well?

Reply:
My reasoning for it originally was that it was since the latest release and the only one which is compatible. But I think it's still safe to have it be ``>=``.